cypress/integration/app.spec.js

```diff
@@ -0,0 +1,12 @@
+describe('Navigation', () => {
+  before(() => {
+    // cy.setupMetamask()
```

Comment:
in different page do we need it, right ?

Reply:
The setupMetamask is apparently broken. However, I think it will be necessary to set it up in the future, so I'll just add the setting.